app/locale/en.coffee

```diff
@@ -990,8 +990,10 @@
     getting_started_3: "You'll see student's progress below as they sign up and join your class."
     additional_resources: "Additional Resources"
     additional_resources_1_pref: "Download/print our"
-    additional_resources_1_mid: "Course 1 Teacher's Guide"
-    additional_resources_1_suff: "explanations and solutions to each level."
+    additional_resources_1_mid: "Course 1"
```

Comment:
When updating an existing locale line in en.coffee, we append ` # {change}`. This helps our tools propagate changes to other languages.

```coffeescript
    additional_resources_1_mid: "Course 1" # {change}
```